federation/module-exposed.cue

```diff
@@ -16,7 +16,7 @@ exposedModule: schema.#Resource & {
 	struct: {
 		id:          schema.IdField
 		handle:      schema.HandleField
-		name: {}
+		name: { sortable: true }
 		node_id: { ident: "nodeID", goType: "uint64", storeIdent: "rel_node" }
 		compose_module_id: { ident: "composeModuleID", goType: "uint64", storeIdent: "rel_compose_module" }
```

Comment:
why in `module-mapping.cue` and not here? or the other way around, why add sorting to the fields that are non-user facing and not used internally in sort?